### ISSUE TITLE
Spotify delay

### ISSUE THREAD

**Issue by AAverin** [beardedspice/beardedspice#579]
Having only web spotify player running, if spotify tab is not selected in chrome, there is a significant (5-8 seconds) delay between pressing a hotkey and action applied in spotify web player.

**Comment by 1ps0** [comment]
How many tabs/windows do you have open? This might be related to a known latency issue that I'm currently working on that affects an overall delay for any site if there's enough tabs open.

**Comment by AAverin** [comment]
@1ps0 I have 2 Chrome instances, about 10 tabs each

**Comment by 1ps0** [comment]
Hmm. This may sound basic, but have you tried restarting the app? same thing?

**Comment by AAverin** [comment]
Yes, sure. Restarted, updated a few times. Will try again, but basically as soon as spotify tab is not active I get a delay

**Comment by 1ps0** [comment]
That gives me a good example case to check against. I'll see if I can replicate it. In the mean time have you checked out the [2.2.0-b1 preview](https://github.com/beardedspice/beardedspice/releases/tag/v2.2.0-b1)?

**Comment by AAverin** [comment]
Preview didn't resolve the issue
